utils/data.py

```diff
@@ -23,13 +47,13 @@ def get_destination(self, no_pm: bool = False):
             return self.context.author
 
     async def send_error_message(self, error):
-        destination = self.get_destination(no_pm=True)
+        destination = self.get_destination(no_pm = True)
```

Comment:
Having space between equals makes my linter (flake8) slap my face for being bad code, pls fix yes.